Sources/Testing/Test+Discovery.swift

```diff
@@ -28,7 +28,7 @@ extension Test {
   /// All available ``Test`` instances in the process, according to the runtime.
   ///
   /// The order of values in this sequence is unspecified.
-  static var all: some Sequence<Test> {
+  static var all: some Collection<Test> {
```

Comment:
What motivates this change?

Reply:
Sorry, that was from a partial implementation and I didn't mean to include it. Will remove.